systemtests/tests/fileset-multiple-include-blocks/testrunner

```diff
@@ -1,15 +1,13 @@
-#!/bin/sh
-
-. scripts/functions
+#!/bin/bash
```

Comment:
Probably this test and the multiple-options-blocks tests can be merged into one test.

Reply:
Yes. As discussed, we first integrate this like this and in a separate PR we try to reduce the number of full tests by cmake FIXTURES.